docs/streams/developer-guide/processor-api.html

```diff
@@ -396,6 +397,21 @@ <h2>
                 </ul>
                 </p>
             </div>
+            <div class="section" id="readonly-state-stores">
+                <span id="streams-developer-guide-state-store-readonly"></span><h3><a class="toc-backref" href="#id12">ReadOnly State Stores</a><a class="headerlink" href="#readonly-state-stores" title="Permalink to this headline"></a></h3>
+                <p>The changelog topic for a fault-tolerant state store can also be used to perform lookups against. It is
+                    crucial to understand that there can be only one writer to this state store, but there can be multiple
+                    readers.</p>
+
+                <p>This allows you to use a readonly state store for looking up reference data managed by another processor
+                    or even another process all together. The messages within the compacted topic may be projected (subset of data extracted)
```

Comment:
> may be projected 

I don't think that works? When we restore the state, we would by-pass the processor and put the bytes from the input topic into the store as-is...

Reply:
yeah, pretty sure eventually we came to that conclusion as well. Too bad though, would have been nice for re-usability purposes.

I will change the description and that the projected stuff out.